scripts/templates/webdriverio.tpl.d.ts

```diff
@@ -633,7 +633,9 @@ declare namespace WebdriverIO {
         // ... browser commands ...
     }
 
-    interface Config extends Options, Omit<WebDriver.Options, "capabilities">, Hooks {}
+    interface Config extends Options, Omit<WebDriver.Options, "capabilities">, Hooks {
+        watch?: boolean
```

Comment:
```suggestion
         /**
         * internal usage only. To run in watch mode see https://webdriver.io/docs/watcher.html
         */
        watch?: never;
```

Reply:
User should not be able to set it in `wdio.conf.js`
![image](https://user-images.githubusercontent.com/25589559/95835214-3b5cc100-0d3e-11eb-8f3e-b43eb96d275b.png)

Reply:
using never type for watch property is failing the execution, we are using `await launcher.onPrepare({ watch: true })` in test file

Reply:
changing `await launcher.onPrepare({ watch: true })`  to `await launcher.onPrepare({ watch: true } as never)` is fixing the failing test. is this ok?

Reply:
Fine for me